### PR TITLE
Alyssa tong contributor spotlight

### PR DESCRIPTION
**Pull request by alyssat**
To be merged on Tuesday, August 13

### CODE REVIEW

File: src/pages/contributors/alyssa-tong.adoc [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+:page-twitter: 
+:page-github: alyssat
+:page-email: 
+:page-image: avatar/alyssa-tong.jpeg
+:page-pronouns: She/Her/Hers
+:page-location: San Diego, California
+:page-firstcommit: 2015
+:page-datepublished: 2024-08-13
+:page-featured: true
+:page-intro: Alyssa Tong is the Jenkins events officer, chair for the Advocacy & Outreach SIG, and org admin for Google Summer of Code. Prior to this, she spent many years directing and producing the Jenkins User conference and Jenkins World. She established Jenkins Area Meetups around the world and started the Jenkins Ambassador program which are now part of the Continuous Delivery Foundation.
```
Comment (krisstern) [inline src/pages/contributors/alyssa-tong.adoc:13]:
```suggestion
:page-intro: Alyssa Tong is the Jenkins events officer, chair for the Advocacy &amp; Outreach SIG, and org admin for Google Summer of Code. Prior to this, she spent many years directing and producing the Jenkins User conference and Jenkins World. She established Jenkins Area Meetups around the world and started the Jenkins Ambassador program which are now part of the Continuous Delivery Foundation.
```

**Review by krisstern — CHANGES_REQUESTED** [review]
May need to trial-and-error to make the "&" sign rendering properly in the intro.

File: src/pages/contributors/alyssa-tong.adoc (hunk 1 above)
Comment (alyssat) [reply src/pages/contributors/alyssa-tong.adoc:13]: Thanks for the suggestion @krisstern, suggestion LGTM

Comment (krisstern) [inline src/pages/contributors/alyssa-tong.adoc:13]:
```suggestion
:page-intro: Alyssa Tong is the Jenkins events officer, chair for the Advocacy {amp} Outreach SIG, and org admin for Google Summer of Code. Prior to this, she spent many years directing and producing the Jenkins User conference and Jenkins World. She established Jenkins Area Meetups around the world and started the Jenkins Ambassador program which are now part of the Continuous Delivery Foundation.
```

**Review by krisstern — APPROVED** [review]
LGTM

**Pull request merged**